lib/web_apis/api_importer.es6.js

```diff
@@ -71,6 +73,22 @@ foam.CLASS({
         });
       },
     },
+    {
+      class: 'Boolean',
+      name: 'toCloudDatastore',
+      documentation: `An optional boolean property. When set true,
+        the importer will import webInterface and browserAPIs to
+        DAO with a rate limit.`,
+      values: false,
```

Comment:
Actually, `Boolean` default is `false, and that API is stable. We can delete this line.